reports/README.md

```diff
@@ -381,7 +425,8 @@ Finally, as for memory usage, the sections that consume the most memory are the
 >
 > Answer:
 
---- question 18 fill here ---
+--- While we had the capability to train our model locally using our CPUs, the necessity for scaling to more intensive compute resources was not immediate. Consequently, we didn't fully explore the potential of Compute Engine for training purposes. For model deployment, hosting our FastAPI server on a Compute Engine VM was a possibility. Yet, we recognized that this option might incur higher costs and offer less scalability compared to serverless alternatives like Cloud Run, which would be our preferred choice for such needs. Cloud Run provides a managed environment that automatically scales based on traffic, making it a more cost-effective and flexible solution for deploying containerized applications like ours.
+ ---
```

Comment:
@notaiax  can you talk a little bit about the VM you used to train the model here? It would be very useful.